CMakeLists.txt

```diff
@@ -111,6 +111,55 @@ if (USE_CCACHE)
     endif()
 endif()
 
+#################################
+# Scafacos Package (optional)   #
+#################################
+
+find_package(PkgConfig)
+if(PkgConfig_FOUND)
+    pkg_check_modules(SCAFACOS IMPORTED_TARGET scafacos)
+    if ( SCAFACOS_FOUND )
+        add_definitions( -DHAVE_CONFIG_H )
+    endif()
+endif()
+
+# Random123
+# - Check for the presence of RANDOM123
+#
+# The following variables are set when RANDOM123 is found:
+#  RANDOM123_FOUND      = Set to true, if all components of RANDOM123 have been found.
+#  RANDOM123_INCLUDES   = Include path for the header files of RANDOM123
+#pkg_check_modules(RANDOM123 IMPORTED_TARGET random123)
```

Comment:
Can we use that?

Reply:
I didn't really compile r123 but specified a path to INCLUDE.
r123 could be loaded as a pkg then we use pkg_check_modules. let me try this tomorrow

Reply:
I installed r123 to /usr/local but pkg_check_* didn't find RANDOM123 as package

Reply:
We should use a real find module like https://gitlab.ebrains.eu/arbor-sim/arbor/-/blob/master/cmake/FindRandom123.cmake

Reply:
> We should use a real find module like https://gitlab.ebrains.eu/arbor-sim/arbor/-/blob/master/cmake/FindRandom123.cmake

Corrected. One can either load preinstalled r123 or compile by specifying absolute path.

Reply:
> Please create a `FindRandom123.cmake` in `cmake/` and replace the code here with `find_package(Random123)`.
Yes, you told so last time. Get back to you soon